crates/bevy_window/src/windows.rs

```diff
@@ -1,36 +1,69 @@
 use super::{Window, WindowId};
 use bevy_utils::HashMap;
 
+/// A collection of [`Window`]s with unique [`WindowId`]s.
 #[derive(Debug, Default)]
 pub struct Windows {
     windows: HashMap<WindowId, Window>,
 }
 
 impl Windows {
+    /// Add the provided [`Window`] to the [`Windows`] resource.
     pub fn add(&mut self, window: Window) {
         self.windows.insert(window.id(), window);
     }
 
+    /// Get a reference to the provided [`WindowId`].
```

Comment:
This doc comment describes:

```rust
fn name(Id: WindowId)->&'static WindowId {
    Box::leak(Box::new(Id))
}
```

I think something like:

```suggestion
    /// Get a reference to the [`Window`] of `id`
```
Is clearer

Similar for below